Reviews/337 -  Royal web wallet and casper connector/1- Development/README.md

```diff
@@ -0,0 +1,394 @@
+$ hardhat run scripts/publish.js
+✅  Published contracts to the subgraph package.
+Done in 7.52s.
+
+```
+
+* Start the frontend on port 3000:
+
+````bash
+[reviewer@rocky8-mae-vm Draw-Bridge]$ sudo yarn start
+Starting the development server...
+
+Browserslist: caniuse-lite is outdated. Please run:
+npx browserslist@latest --update-db
+
+Why you should do it regularly:
+https://github.com/browserslist/browserslist#browsers-data-updating
+
+
+````
+
+* Connect to the frontend:
+
+The reviewer was able to connect to the frontend 
+
+![frontend](assets/front_end.png "frontend")
+
+The reviewer tried to run the test scenario given in the README file :
+
+````bash
+1. Connect Metamask
+2. Connect Casper Signer
+3. Grab ETH Faucet (for testent)
+4. Grab CSPR Faucet (for testent) 
+5. Enter amount
+6. Push Exchane
+7. If you are Supervisor:
+7.1 enter CSPR privkey for order processing
+7.2 Verify tx and push Approve
+7.3 Push Cancel if not sure and back ETH manually
+````
+
+
+1- Connect Metamask
+
+The reviewer was able to connect to the testnet ETH network (Rinkeby) via Metamask.
+
+* Set the ETH network to Rinkeby
+![metamask testnet](assets/metamask_testnet_1.png "metamask_1")
+
+* Connect wallet using metamask
+![metamask testnet](assets/metamask_testnet_2.png "metamask_2")
+
+![metamask testnet](assets/metamask_testnet_3.png "metamask_2")
+
+The informations about the current connected test ETH wallet and  the available test ETH balance seem to be wrong.
+The reviewer thinks that connecting to ETH networks via metamask  does not work properly in the application.
+But he cannot confirm that since there are no errors indicating it on the web console.
+The page shows an error message (Can't load ethereum contract. Please check configuration).This error is most likely due to the missing variables we could not set above.
+
+**The OP is strongly recommended to give a more detailled step-by-step guide on how to use the application**
+
+2- Connect Casper Signer
+
+The reviewer was able to connect to the Casper testnet via CasperSigner. The public key of Caper signer wallet was shown on the page. 
+**The balance of the connected account (1000 test cspr) was not displayed on the page 
+and an error keeps  being thrown in the web console (Chrome).**
+
+```console
+Access to fetch at 'http://138.201.54.44:7777/rpc' from origin 'http://localhost:3000' 
+has been blocked by CORS policy: Response to preflight request doesn't pass access control 
+check: No 'Access-Control-Allow-Origin' header is present on the requested resource. 
+If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the 
+resource with CORS disabled.
+```
+
+
+![Casper testnet](assets/casper_testnet.png "metamask")
+
+3. Grab ETH Faucet (for testnet) 
+
+The reviewer could not use his own testnet ETH wallet  on the Rinkeby ETH test network.
```

Comment:
Why? Rinkeby faucet seems functional.

Reply:
In fact, i could connect to Rinkeby network with metamask, but the page did not display my wallet address but a random wallet one..

Reply:
Could you please paraphrase this part to reflect that?

Reply:
changed to this : The reviewer could not use his own testnet ETH wallet on the Rinkeby ETH test network. After connecting to the Rinkeby network via metamask, the address displayed on the page (top right) seems to be a random one and does not correspond to the address of the reviewer's metamask wallet.

Is that ok?

Reply:
Perfect. Thank you!